src/components/Firestore/Requests/RequestDetails/index.tsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+interface PropsFromParentComponent {
+  requestId?: string;
+}
+export type Props = PropsFromParentComponent;
```

Comment:
Small nit: Avoid using `type`.

Instead, prefer

```ts
export interface Props extends PropsFromParentComponent {}
```

Reply:
I didn't know this approach was preferred for React Props, thank you!